supporter-product-data/src/main/scala/com/gu/services/DynamoDBService.scala

```diff
@@ -44,26 +57,43 @@ class DynamoDBService(client: DynamoDbAsyncClient, tableName: String) {
       .expressionAttributeValues(attributeValues)
       .build
 
-    client.updateItem(updateItemRequest).toScala.recoverWith {
-      case cex: CompletionException => Future.failed(cex.getCause)
-    }
+    client.updateItem(updateItemRequest).toScala
   }
 
-  def getAdjustedTermEndDate(date: LocalDate) =
-    date.plusDays(1) // This is to avoid problems with timezones, as renewals are created early morning Pacific Time
+  def asEpochSecond(date: LocalDate) =
+    date
       .atStartOfDay
       .toEpochSecond(ZoneOffset.UTC)
       .toString
+
+  def asIso(date: LocalDate) =
+    date.format(DateTimeFormatter.ISO_LOCAL_DATE)
 }
 
-object DynamoDBService{
+object DynamoDBService {
   lazy val CredentialsProvider = AwsCredentialsProviderChain.builder.credentialsProviders(
     ProfileCredentialsProvider.builder.profileName(ProfileName).build,
     InstanceProfileCredentialsProvider.builder.asyncCredentialUpdateEnabled(false).build,
     EnvironmentVariableCredentialsProvider.create()
   ).build
 
-  val dynamoDBClient = DynamoDbAsyncClient.builder
+  val clientOverrideConfiguration = ClientOverrideConfiguration.builder.retryPolicy(
```

Comment:
This config makes sure we keep on retrying updates until they succeed, the default retry policy gives up quite quickly so we can end up failing because of a `ProvisionedThroughputExceededException` before the autoscaling kicks in.